aws/amazon-ec2-metadata-mock
# update license check

Issue #, if available:

* `THIRD_PARTY_LICENSES` file must be updated manually
* `make license-test` is slow

Description of changes:

* Use `go-licenses` to generate third-party licenses file
* Use `go-licenses` to speed up `make license-test`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
